packages/lesswrong/components/posts/PostsPage/BestOfLessWrong/BestOfLessWrongAnnouncement.tsx

```diff
@@ -221,8 +267,13 @@ const BestOfLessWrongAnnouncement = () => {
   return (
     <AnalyticsContext pageSectionContext="bestOfLessWrongAnnouncement">
       <SingleColumnSection>
-        <div className={classes.title}>
-          Best of LessWrong {REVIEW_YEAR}
+        <div className={classes.titleContainer}>
+          <Link to={`/bestoflesswrong`} className={classes.title}>
```

Comment:
Should this link contain something in the path for the indicated year?  (I'm a little confused by this link and the next link pointing to the exact same place.)

Reply:
It autodirects to the current year so it doesn't really matter.